MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -478,10 +482,20 @@ - (void)deleteBufferedLogWithInternalId:(NSString *)internalId {
     for (auto it = msCrashesLogBuffer.begin(), end = msCrashesLogBuffer.end(); it != end; ++it) {
       NSString *bufferId = [NSString stringWithCString:it->internalId.c_str() encoding:NSUTF8StringEncoding];
       if (bufferId && bufferId.length > 0 && [bufferId isEqualToString:internalId]) {
-        MSLogVerbose([MSCrashes logTag], @"Deleting item from buffer with id %@", internalId);
+        MSLogVerbose([MSCrashes logTag], @"Deleting a log from buffer with id %@", internalId);
         it->buffer = [@"" cStringUsingEncoding:NSUTF8StringEncoding];
         it->timestamp = [@"" cStringUsingEncoding:NSUTF8StringEncoding];
         it->internalId = [@"" cStringUsingEncoding:NSUTF8StringEncoding];
+        if (writeBufferTaskStarted) {
```

Comment:
Is it possible to unit test this method? To check the behavior with writeBufferTaskStarted set to NO and then set to YES?